app/components/publications/publication/publication-activities/publications-info-panel.js

```diff
@@ -0,0 +1,69 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+/**
+ * @argument {publicationSubcase}
+ * @argument {PublicationFlow}
+ * @argument {isFinal} if publication status is in final state
+ */
+export default class PublicationsPublicationPublicationActivitiesPublicationInfoPanel extends Component {
+  @service store;
+
+  @tracked isEditing = false;
+  @tracked publicationStatus;
+  @tracked publicationModes;
+
+  @tracked publicationMode;
+  @tracked targetEndDate;
+
+
+  constructor() {
+    super(...arguments);
+    this.publicationModes = this.store.peekAll('publication-mode').sortBy('position');
+    this.initFields();
+  }
+
+  initFields() {
+    this.targetEndDate = this.args.publicationSubcase.targetEndDate;
+    this.publicationMode = this.args.publicationFlow.mode;
```

Comment:
Dit is een async-relatie dus wil je `await`-en. Al kan je ook hier rechtstreeks op `@publicationFlow.mode` werken i.p.v. op een copy. Om dat te rollbacken wanneer de edit-mode afgesloten wordt zonder opslaan, kan je [`this.args.publicationFlow.belongsTo('mode').reload()`](https://api.emberjs.com/ember-data/3.25/classes/BelongsToReference/methods/reload?anchor=reload) gebruiken.